core/src/main/scala/org/http4s/QueryParam.scala

```diff
@@ -61,6 +99,8 @@ object QueryParamEncoder {
   /** summon an implicit [[QueryParamEncoder]] */
   def apply[T](implicit ev: QueryParamEncoder[T]): QueryParamEncoder[T] = ev
 
+  def fromCodec[T](implicit ev: QueryParamCodec[T]): QueryParamEncoder[T] = ev
```

Comment:
I don't think circe has these.  Thinking about how these might clash.

Reply:
I thought this was implicit when I first read it.  I feel better about it on second look.